src/standalone/Snow/snow_parameterizations.jl

```diff
@@ -8,7 +8,20 @@ export snow_surface_temperature,
     runoff_timescale,
     compute_water_runoff,
     energy_from_q_l_and_swe,
-    energy_from_T_and_swe
+    energy_from_T_and_swe,
+    scf
+
+"""
+    scf(x::FT; α = FT(1e-3))::FT where {FT}
+
+Returns the snow cover fraction, assuming it is a heaviside
+function at 1e-3 meters.
+
+In the future we can play around with other forms.
+"""
+function scf(x::FT; α = FT(1e-3))::FT where {FT}
```

Comment:
`scf` isn't a very informative name - could we use `snow_cover_fraction` instead?